src/frontend/components/SøknadsSteg/Oppsummering/Oppsummering.tsx

```diff
@@ -1,12 +1,87 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
+
+import { Alpha3Code } from 'i18n-iso-countries';
+import { useIntl } from 'react-intl';
+
+import { Element, Normaltekst } from 'nav-frontend-typografi';
 
 import { useApp } from '../../../context/AppContext';
+import { landkodeTilSpråk } from '../../../utils/person';
+import SpråkTekst from '../../Felleskomponenter/SpråkTekst/SpråkTekst';
+import Steg from '../../Felleskomponenter/Steg/Steg';
+import Oppsummeringsbolk from './Oppsummeringsbolk';
+
+interface IOppsummeringsFeltProps {
+    tittel: ReactNode;
+    søknadsvar?: string;
+}
+
+const OppsummeringFelt: React.FC<IOppsummeringsFeltProps> = ({ tittel, søknadsvar }) => {
+    return (
+        <>
+            <Element>{tittel}</Element>
+            <Normaltekst>
+                <SpråkTekst id={søknadsvar} />
+            </Normaltekst>
+        </>
+    );
+};
 
 const Oppsummering: React.FC = () => {
+    const intl = useIntl();
+
     const { søknad } = useApp();
     console.log(søknad);
 
-    return <div>Her kommer det en oppsummering!</div>;
+    return (
+        <Steg tittel={<SpråkTekst id={'ombarnadine.tittel'} />}>
+            <Oppsummeringsbolk tittel={'oppsummering.omdeg-tittel'}>
+                <OppsummeringFelt
+                    tittel={<SpråkTekst id={'omdeg.personopplysninger.fødselsnummer'} />}
+                    søknadsvar={søknad.søker.ident}
+                />
+                <OppsummeringFelt
+                    tittel={<SpråkTekst id={'omdeg.personopplysninger.statsborgerskap'} />}
+                    søknadsvar={søknad.søker.statsborgerskap
+                        .map((statsborgerskap: { landkode: Alpha3Code }) =>
+                            landkodeTilSpråk(statsborgerskap.landkode, intl.locale)
+                        )
+                        .join(', ')}
+                />
+                <OppsummeringFelt
+                    tittel={<SpråkTekst id={'omdeg.personopplysninger.sivilstatus'} />}
+                    søknadsvar={søknad.søker.sivilstand.type}
+                />
+                <OppsummeringFelt
+                    tittel={<SpråkTekst id={'omdeg.telefon.spm'} />}
+                    søknadsvar={søknad.søker.telefonnummer.svar}
+                />
+                <OppsummeringFelt
+                    tittel={<SpråkTekst id={'omdeg.opphold-i-norge.spm'} />}
+                    søknadsvar={søknad.søker.oppholderSegINorge.svar}
+                />
+                {søknad.søker.oppholdsland.svar !== undefined && (
```

Comment:
her ville jeg enten sjekket på søknad.søker.oppholderSegINorge.svar === ESvar.Nei (det som er felles for begge) eller sjekket om dato også er var tom (sjekket de hver for seg)😊  nå blir dato på en måte litt avhengig av landet

Reply:
Sant, tar det med i neste PR, så bare merger jeg dette nå 😊